Algorithms/Search/RecursiveBinarySearcher.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Algorithms.Searches
+{
+    /// <summary>
+    /// TODO.
```

Comment:
Codacy found an issue: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6286772)